lib/rake/test.rake

```diff
@@ -122,6 +122,7 @@ namespace :test do
           Dir["#{fixture_path}/{**,*}/*.yml"].
             push(dashboard_dir('db/schema.rb')).
             push(dashboard_dir('config/videos.csv')).
+            push(dashboard_dir('schools.tsv')).
```

Comment:
I thought the one that is modified is in the fixture path? Should this be `push("#{fixture_path"}/schools.tsv")`?

Reply:
Ohhh I was misreading this. Yes- will update